src/Framework/LazyFormattedBuildEventArgs.cs

```diff
@@ -199,45 +199,6 @@ private static string FormatString(CultureInfo culture, string unformatted, para
             if ((args?.Length > 0))
             {
 #if DEBUG
-
-#if VALIDATERESOURCESTRINGS
```

Comment:
Nit: I'd ideally like cleanup like this in a separate commit so we can easily separate it out from the more interesting work in the PR.

Reply:
Understand